cl/search/types.py

```diff
@@ -85,3 +99,16 @@
 SaveDocumentResponseType = tuple[str, ESDictDocument]
 
 SearchAlertHitType = tuple[Alert, str, list[ESDictDocument], int]
+
+
+class EventTable(Enum):
+    DOCKET = "search.Docket"
+    DOCKET_ENTRY = "search.DocketEntry"
+    RECAP_DOCUMENT = "search.RECAPDocument"
+
+    @classmethod
+    def get_member(cls, value):
+        for member in cls:
+            if member.value == value:
+                return member
+        return None
```

Comment:
We can simplify our code by using the `StrEnum` class directly instead of adding the `get_member` method. This will eliminate the need for a separate method to retrieve one of the members when a string is provided. By using `StrEnum`, we can directly compare the value provided by the user with the members of the class.